products/mlengine/api.yaml

```diff
@@ -235,4 +235,338 @@ objects:
+            name: 'workerCount'
+            description: | 
+              The number of worker replicas to use for the training job. Each replica in the cluster will be of the type 
+              specified in workerType
+            required: false
+          - !ruby/object:Api::Type::Integer
+            name: 'parameterServerCount'
+            description: | 
+              The number of parameter server replicas to use for the training job. Each replica in the cluster will be 
+              of the type specified in parameterServerType.
+            required: false
+          - !ruby/object:Api::Type::Array
+            name: 'packageUris'
+            description: |
+              The Google Cloud Storage location of the packages with the training program and any additional dependencies.
+               The maximum number of package URIs is 100.
+            required: true
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::String
+            name: 'pythonModule'
+            description: The Python module name to run after installing the packages.
+            required: true
+          - !ruby/object:Api::Type::Array
+            name: 'args'
+            description: Command line arguments to pass to the program.
+            required: false
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::String
+            name: 'region'
+            description: The Google Compute Engine region to run the training job in. See the available regions for AI Platform services.
+            required: true 
+          - !ruby/object:Api::Type::String
+            name: 'jobDir'
+            description: |
+               A Google Cloud Storage path in which to store training outputs and other data needed for training. This path is passed 
+               to your TensorFlow program as the '--job-dir' command-line argument. The benefit of specifying this field is that Cloud 
+               ML validates the path for use in training.
+            required: false 
+          - !ruby/object:Api::Type::String
+            name: 'runtimeVersion'
+            description: | 
+              The AI Platform runtime version to use for training. If not set, AI Platform uses the default stable version, 1.0. For more 
+              information, see the runtime version list and how to manage runtime versions.
+            required: false 
+          - !ruby/object:Api::Type::String
+            name: 'pythonVersion'
+            description: |
+              The version of Python used in training. If not set, the default version is '2.7'. Python '3.5' is available when 
+              runtimeVersion is set to '1.4' and above. Python '2.7' works with all supported runtime versions.
+            required: false 
+      ## Training output start here      
+      - !ruby/object:Api::Type::NestedObject
+        name: 'trainingOutput'
+        description: The current training job result.
+        output: true
```

Comment:
This doesn't currently propagate through its nested fields (it probably should), so you'll want to set `output: true` on all of the properties inside of it.

Reply:
ok